### ISSUE TITLE
`Callable` with `ParamSpec` is reported as partially unknown when used as variable or return type annotations

### ISSUE THREAD

**Issue by bryanforbes** [microsoft/pyright#2665]
**Describe the bug**
Given the following code:
```python
from __future__ import annotations

from typing import Any, Callable
from typing_extensions import ParamSpec

P = ParamSpec('P')

Handler = Callable[P, Any]


def foo() -> Handler[[list[str], int]]:
    def handler(strings: list[str], number: int) -> Any:
        return {}

    return handler


f: Handler[[list[str], int]] = foo()
```

The `Handler` annotations at lines 11 and 18 are reported as partially unknown.

**Expected behavior**
No error should be reported and the `Handler` should be bound with `(list[str], int)`.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool?
Command line

Which version?
1.1.193

**Comment by erictraut** [comment]
This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.194, which I just published. It will also be included in the next release of pylance.
